src/Podium/Request.ts

```diff
@@ -175,7 +175,7 @@ export class Request {
         if (Token.getInstance().GetToken()) {
             return {
                 'Accept-Language': this.GetLocale(),
-                'Authentication': Token.getInstance().GetToken(),
+                'Authorization': Token.getInstance().GetToken(),
```

Comment:
Has this header property changed for every request in the API?

Reply:
Yes, so JWT requires 'Authorization' instead of 'Authentication' any more